src/request/request.service.ts

```diff
@@ -98,16 +141,43 @@ export class RequestService {
 
     // 1 because to get team name from description
     const teamName = input.description.split(' ')[1];
+    const player = input.description.split(' ')[4];
     const team = await this.teamService.getTeamByName(teamName);
     if (!team) {
       throw new HttpException(NO_SUCH_TEAM, HttpStatus.BAD_REQUEST);
     }
 
+    const user = await this.userService.getUserById(input.from)
+
     // add team for user
-    await this.userService.addToTeam(team, input.from);  // если true? то тогда делать метку approve .................................
+    try {
+      await this.userService.addToTeam(team, input.from);  // если true? то тогда делать метку approve .................................
```

Comment:
don't understand the question

Reply:
this comment was for the future. Because if the add operation fails, then the request type should remain the same. Now impoved